prerequisites/install-functions/build_opencoarrays.sh

```diff
@@ -13,15 +13,15 @@ build_opencoarrays()
   if [[ -z ${MPIFC:-} || -z ${MPICC:-} ]]; then
     emergency "build_opencoarrays.sh: empty \${MPIFC}=${MPIFC:-} or \${MPICC}=${MPICC:-}"
   fi
-  MPIFC_show=`$MPIFC -show`
-  MPICC_show=`$MPICC -show`
-  if [[ ${MPIFC_show} != *gfortran* || ${MPICC_show} != *gcc* ]]; then
-    emergency "build_opencoarrays.sh: MPI doesn't wrap gfortran/gcc: \${MPIFC_show}=${MPIFC_show}, \${MPICC_show}=${MPICC_show}"
-  fi 
+  MPIFC_show=($($MPIFC -show))
+  MPICC_show=($($MPICC -show))
+  if [[ "${MPIFC_show[0]}" != *gfortran* || "${MPICC_show[0]}" != *gcc* ]]; then
```

Comment:
We may want to add some more robust handling in the event that the installation location has spaces in the path: I'm not sure if the MPI wrappers will properly quote and/or escape these cases... It's tricky too, because it's possible for the compiler in the wrapper to just pull from the path (e.g., `gcc`) or be a full or relative path (e.g., `/usr/local/bin/gcc`)... One idea is to use `which`, `hash`, or `type` to determine if array element 0 is a valid executable, and if not, try to add the 1st element, test again, add 2nd etc. to handle spaces... 

Another thought: parse up until the first occurrence of `[:blank:]-` which will indicate a flag... this would be fine except, perhaps for cases where the compiler has MPI built in, like Cray... but I think there is some exceptional logic to handle this anyway.